torchvision/transforms/transforms.py

```diff
@@ -909,6 +912,7 @@ def _check_input(self, value, name, center=1, bound=(0, float('inf')), clip_firs
         return value
 
     @staticmethod
+    @torch.jit.unused
     def get_params(brightness, contrast, saturation, hue):
```

Comment:
@clemkoa looks like `get_params` becomes unused, why did we keep it ?

Reply:
For backwards-compatibility

Reply:
I can give a shot to that as I'm working on `F.adjust_hue` and it is used by ColorJitter...

Reply:
Sounds good to me, but can you do it in a separate PR which only does this?

Reply:
No problems, can do it in a separate PR